assignments/callbacks.js

```diff
@@ -26,30 +26,64 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 
 function getLength(arr, cb) {
+ return cb(arr.length);
   // getLength passes the length of the array into the callback.
 }
 
+getLength(items, function(length){
+console.log(length);
+});
+
 function last(arr, cb) {
+  return cb(arr[arr.length-1]);
   // last passes the last item of the array into the callback.
 }
 
+last(items, function(lastitem){
+console.log(lastitem);
+});
+
 function sumNums(x, y, cb) {
+  return cb(x + y);
   // sumNums adds two numbers (x, y) and passes the result to the callback.
 }
 
+sumNums(5, 3, function(add){
+console.log(add);
+});
+
 function multiplyNums(x, y, cb) {
+  return cb(x * y);
   // multiplyNums multiplies two numbers and passes the result to the callback.
 }
 
+multiplyNums(5, 8, function(multiply){
+console.log(multiply);
+})
+
 function contains(item, list, cb) {
+  if (list.includes(item)) {
+    return cb(true);
+  }else{
+    return cb(false);
+  }
   // contains checks if an item is present inside of the given array/list.
   // Pass true to the callback if it is, otherwise pass false.
 }
 
+contains('Notebook', items, function(contain){
```

Comment:
instead of `function)contain{console.log(contain)}`, you can now use `console.log`